### PR TITLE
update entra-previews links

### PR DESCRIPTION
**Pull request by BenBagBag**
## Purpose

* Updates link/link names for old entra-preview links. NB: this **does not** update other wording. Expect sentences like "get a CIAM tenant and use this Entra tutorial <link>". This will hopefully prevent messy merge issues later on.

## Does this introduce a breaking change?

```
[ ] Yes
[ ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

File: 1-Authentication/0-sign-in-vanillajs/README.md
Comment (iambmelt) [inline 1-Authentication/0-sign-in-vanillajs/README.md:66]: I'll defer to @kengaderdus to say whether "external Microsoft Entra ID tenant" or "Microsoft Entra External ID tenant" is preferred

Comment (iambmelt) [reply 1-Authentication/0-sign-in-vanillajs/README.md:66]: Oh nevermind, I see Katherine asked for it this way 👍

**Review by iambmelt — APPROVED** [review]
LGTM, let's also get a review from @kengaderdus before merging

File: 1-Authentication/0-sign-in-vanillajs/README.md
Comment (kengaderdus) [inline 1-Authentication/0-sign-in-vanillajs/README.md:39]:
```suggestion
This sample demonstrates a vanilla JavaScript single-page application (SPA) that lets users authenticate to [Microsoft Entra External ID](https://learn.microsoft.com/entra/external-id/external-identities-overview) using the [Microsoft Authentication Library for JavaScript](https://github.com/AzureAD/microsoft-authentication-library-for-js) (MSAL.js). 
```

Comment (kengaderdus) [reply 1-Authentication/0-sign-in-vanillajs/README.md:39]: @BenBagBag 

For links, remove the locale. For examples use `https://learn.microsoft.com/entra/external-id/external-identities-overview` instead of `https://learn.microsoft.com/en-us/entra/external-id/external-identities-overview`.

File: 1-Authentication/0-sign-in-vanillajs/README.md
Comment (kengaderdus) [inline 1-Authentication/0-sign-in-vanillajs/README.md:66]:
```suggestion
* An external tenant. For more information, see: [Create a new tenant in Microsoft Entra ID](https://learn.microsoft.com/entra/external-id/customers/how-to-create-external-tenant-portal)
```

Comment (kengaderdus) [reply 1-Authentication/0-sign-in-vanillajs/README.md:66]: Just say _external tenant_, here and in all other instances as you already told the user the product name, that's _Microsoft Entra External ID._

**Review by kengaderdus — CHANGES_REQUESTED** [review]
@BenBagBag
Here's a summary of my comments:
- In the links, remove the locale.
- Refer to the CIAM tenant as external tenant, but not `CIAM tenant` or `external Microsoft Entra ID tenant` or `Microsoft Entra External ID tenant`

**Comment by BenBagBag** [comment]
@kengaderdus I have updated the links.

For the wording, your suggestion does not match up with the request in the original ticket (I split the link changes out into a separate ticket, hence different PR): https://identitydivision.visualstudio.com/Engineering/_workitems/edit/2960843?src=WorkItemMention&src-action=artifact_link

Can you consult with @kleggMS and @henrymbuguakiarie to get agreement on wording? Thank you!

**Comment by kengaderdus** [comment]
@BenBagBag 

What's not matching? Can you mention one instance or example.

What I said is, if you refer to the product name, use _Microsoft Entra External ID_, if you're referring to the CIAM tenant, use _external tenant_.

**Comment by BenBagBag** [comment]
@kengaderdus Ah, that makes sense, thanks for clarifying. I've updated it, please re-review.

CC: @kleggMS

File: 1-Authentication/0-sign-in-vanillajs/README.md
Comment (kengaderdus) [inline 1-Authentication/0-sign-in-vanillajs/README.md:39]:
```suggestion
This sample demonstrates a vanilla JavaScript single-page application (SPA) that lets users authenticate to [Microsoft Entra External ID](https://learn.microsoft.com/entra/external-id/customers/how-to-create-external-tenant-portal) using the [Microsoft Authentication Library for JavaScript](https://github.com/AzureAD/microsoft-authentication-library-for-js) (MSAL.js). 
```

File: 1-Authentication/0-sign-in-vanillajs/README.md
Comment (kengaderdus) [inline 1-Authentication/0-sign-in-vanillajs/README.md:66]:
```suggestion
* An external tenant. For more information, see: [Create a new external tenant in Microsoft Entra ID](https://learn.microsoft.com/entra/external-id/customers/how-to-create-external-tenant-portal)
```

File: 1-Authentication/1-sign-in-react/README.md
Comment (kengaderdus) [inline 1-Authentication/1-sign-in-react/README.md:65]:
```suggestion
 * An external tenant. For more information, see: [Create a new external tenant in Microsoft Entra ID]https://learn.microsoft.com/entra/external-id/customers/how-to-create-external-tenant-portal)
```

File: 1-Authentication/2-sign-in-angular/README.md
Comment (kengaderdus) [inline 1-Authentication/2-sign-in-angular/README.md:65]:
```suggestion
* An external tenant. For more information, see: [Create a new external tenant in Microsoft Entra ID](https://learn.microsoft.com/entra/external-id/customers/how-to-create-external-tenant-portal)
```

File: 1-Authentication/3-sign-in-electron/README.md
Comment (kengaderdus) [inline 1-Authentication/3-sign-in-electron/README.md:71]:
```suggestion
 * An external tenant. For more information, see: [Create a new external tenant in Microsoft Entra ID](https://learn.microsoft.com/entra/external-id/customers/how-to-create-external-tenant-portal)
```

File: 1-Authentication/4-sign-in-device-code/README.md
Comment (kengaderdus) [inline 1-Authentication/4-sign-in-device-code/README.md:57]:
```suggestion
 * An external tenant. For more information, see: [Create a new external tenant in Microsoft Entra ID](https://learn.microsoft.com/entra/external-id/customers/how-to-create-external-tenant-portal)
```

File: 1-Authentication/6-sign-in-node-cli-app/README.md
Comment (kengaderdus) [inline 1-Authentication/6-sign-in-node-cli-app/README.md:64]:
```suggestion
 * An external tenant. For more information, see: [Create a new external tenant in Microsoft Entra ID](https://learn.microsoft.com/entra/external-id/customers/how-to-create-external-tenant-portal)
```

File: 2-Authorization/0-call-api-vanillajs/README.md
Comment (kengaderdus) [inline 2-Authorization/0-call-api-vanillajs/README.md:73]:
```suggestion
 * An external tenant. For more information, see: [Create a new external tenant in Microsoft Entra ID](https://learn.microsoft.com/entra/external-id/customers/how-to-create-external-tenant-portal)
```

File: 2-Authorization/1-call-api-react/README.md
Comment (kengaderdus) [inline 2-Authorization/1-call-api-react/README.md:70]:
```suggestion
 * An external tenant. For more information, see: [Create a new external tenant in Microsoft Entra ID](https://learn.microsoft.com/entra/external-id/customers/how-to-create-external-tenant-portal)
```

File: 2-Authorization/2-call-api-angular/README.md
Comment (kengaderdus) [inline 2-Authorization/2-call-api-angular/README.md:72]:
```suggestion
* An external tenant. For more information, see: [Create a new external tenant in Microsoft Entra ID](https://learn.microsoft.com/entra/external-id/customers/how-to-create-external-tenant-portal)
```

File: 2-Authorization/3-call-api-node-daemon/README.md
Comment (kengaderdus) [inline 2-Authorization/3-call-api-node-daemon/README.md:70]:
```suggestion
* An external tenant. For more information, see: [Create a new external tenant in Microsoft Entra ID](https://learn.microsoft.com/entra/external-id/customers/how-to-create-external-tenant-portal)
```

File: 2-Authorization/4-call-api-express/README.md
Comment (kengaderdus) [inline 2-Authorization/4-call-api-express/README.md:70]:
```suggestion
* An external tenant. For more information, see: [Create a new external tenant in Microsoft Entra ID](https://learn.microsoft.com/entra/external-id/customers/how-to-create-external-tenant-portal)
```

**Review by kengaderdus — CHANGES_REQUESTED** [review]
@BenBagBag 
I noticed the link for creating an external tenant was pointing to the wrong doc. So, I've update all  instances of instances of An external tenant. For more information, see: [Create a new tenant in Microsoft Entra ID](https://learn.microsoft.com/entra/fundamentals/create-new-tenant) to  An external tenant. For more information, see: [Create a new external tenant in Microsoft Entra ID](https://learn.microsoft.com/entra/external-id/customers/how-to-create-external-tenant-portal).

Once this change is made, then everything is good for me.

**Review by kengaderdus — APPROVED** [review]

**Review by iambmelt — APPROVED** [review]

**Pull request merged**
